.changeset/mighty-beds-laugh.md

```diff
@@ -0,0 +1,3 @@
+---
+"@khanacademy/perseus": patch
+---
```

Comment:
IMO changesets either need to be empty (`yarn changeset --empty`) or should have a description. If these are all test changes, I think an empty changeset would be appropriate.

Reply:
Apologies! Forgot to update this PR after learning about empty changesets with my other PR. Fixed!